src/main/java/uk/gov/hmcts/divorce/citizen/notification/ApplicationOutstandingActionNotification.java

```diff
@@ -99,25 +96,30 @@ private Map<String, String> applicant1TemplateVars(final CaseData caseData, fina
 
     private Map<String, String> applicant2TemplateVars(final CaseData caseData, final Long id) {
         Map<String, String> templateVars = commonContent.mainTemplateVars(caseData, id, caseData.getApplicant2(), caseData.getApplicant1());
-        templateVars.putAll(courtDocumentDetails(caseData, templateVars.get(APPLICATION_REFERENCE), templateVars.get(COURT_EMAIL)));
+        templateVars.putAll(courtDocumentDetails(caseData));
         templateVars.putAll(serveAnotherWayTemplateVars(false, caseData));
         return templateVars;
     }
 
     private Map<String, String> serveAnotherWayTemplateVars(boolean soleServingAnotherWay, CaseData caseData) {
         Map<String, String> templateVars = new HashMap<>();
 
+        boolean servedAnotherWay = soleServingAnotherWay && caseData.isDivorce();
+        boolean languagePreferenceWelsh = WELSH == caseData.getApplicant1().getLanguagePreference();
+        String welshDivorceOrDissolution = caseData.isDivorce() ? DIVORCE_WELSH : "";
+        String divorceOrDissolution = caseData.isDivorce() ? DIVORCE : "";
+        String partner = soleServingAnotherWay ? commonContent.getPartner(caseData, caseData.getApplicant2(),
+                caseData.getApplicant1().getLanguagePreference()) : "";
+
         templateVars.put(PAPERS_SERVED_ANOTHER_WAY, soleServingAnotherWay ? YES : NO);
-        templateVars.put(DIVORCE_SERVED_ANOTHER_WAY, soleServingAnotherWay && caseData.isDivorce() ? YES : NO);
-        templateVars.put(SERVE_WIFE_ANOTHER_WAY,
-            soleServingAnotherWay && caseData.isDivorce() && caseData.getApplicant2().getGender().equals(Gender.FEMALE) ? YES : NO);
-        templateVars.put(SERVE_HUSBAND_ANOTHER_WAY,
-            soleServingAnotherWay && caseData.isDivorce() && caseData.getApplicant2().getGender().equals(Gender.MALE) ? YES : NO);
+        templateVars.put(DIVORCE_OR_DISSOLUTION, languagePreferenceWelsh ? welshDivorceOrDissolution : divorceOrDissolution);
```

Comment:
If the nitpick on line 110 is made:
```suggestion
        templateVars.put(DIVORCE_OR_DISSOLUTION, divorceOrDissolution);
```